### DISCUSSION TITLE
What does the "Force Security" do?

### DISCUSSION THREAD

**Discussion by miguelarios** [zwave-js/zwave-js-ui#1737] [Q&A] [answered]
I have a Yale lock that says it supports S2 Access Control yet I paired it with ZwaveJS2MQTT and i got S0, should I be selecting "Force Security"?

https://products.z-wavealliance.org/products/4092?selectedFrequencyId=2

**Comment by miguelarios** [comment]
OK i just tried it with Force Security and nope it still got added with S0_Legacy. Why would this be the case?

**Comment by miguelarios** [comment, marked as answer]
OK just read I have the ZW2 not the ZW3 :/

**Reply by robertsLando** [reply]
@miguelarios Did you added all S2 keys in zwave settings?

**Reply by miguelarios** [reply]
> @miguelarios Did you added all S2 keys in zwave settings?

Yup! I created a random key for each. I don’t know if it is because the YRD226-ZW2 is first gen ZWave Plus so when I looked at the zwave alliance page it doesn't seem to have S2 while YRD226-ZW3 does and was just released this year according to the zwave alliance page yet I can't find it online.
